### PR TITLE
Added a WYSIWYG HTML Editor to all Template

### PR DESCRIPTION
**Pull request by Tahirih-Natang**
### Description
Added a ckeditor to all relevant template across the entire project. Also updated requirements.txt for all new dependencies
Fixes # The Issue was not officially opened. I thought it will be an enhancement to the current app

### Type of Change:
- Code
- User Interface
  

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

### CODE REVIEW

**Comment by Tahirih-Natang** [comment]
@medhach , @MeepyMay , @Nerdylicious Please review this pull request and let me get your feedbacks

**Comment by muddlebee** [comment]
everything looks good. 👍 Screen shots?

File: malaria_web/tests.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        form = PostForm(data={'title_post': "post", 'description_post': "Description", 'link_post': "url", 'photo': "image"})
+        self.assertTrue(form.is_valid())
+
+    # Invalid Form Data
+    def test_PostForm_invalid(self):
+        form = PostForm(data={'title_post': "", 'description_post': "mp", 'link_post': "mp", 'photo': ""})
+        self.assertFalse(form.is_valid())
+
+class View_Test(TestCase):
+    #This comment is just to replace trailing whitespace
```
Comment (medhach) [inline malaria_web/tests.py:27]: You can remove the line instead of adding the comment. :)

File: pcsa_GHN/tests.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    def test_ghn_ContactForm_valid(self):
+        contact = ContactForm(data={'office_name': "valid name of office branch", 'contact_number': "679866532"})
+        self.assertTrue(contact.is_valid())
+
+    # Invalid Form Data for Contact Form
+    def test_ghn_ContactForm_invalid(self):
+        contact = ContactForm(data={'office_name': "null", 'contact_number': "string"})
+        self.assertFalse(contact.is_valid())
+
+    # def test_ghn_contactForm_type(self):
```
Comment (medhach) [inline pcsa_GHN/tests.py:33]: Can you explain why this test is commented?

File: templates/malaria/create_post.html [hunk 3]
```diff
@@ -1,6 +1,4 @@
-{% include "infohub/index.html" %}
-
-{% block content %}
+{% include "infohub/index.html" %} {% block content %}
```
Comment (medhach) [inline templates/malaria/create_post.html:1]: Can you explain why you made this change?

File: templates/malaria/create_post.html [hunk 4]
```diff
@@ -78,5 +77,4 @@ <h2 class="text-center">New Post</h2>
         </form>
     </div>
 </div>
-<br><br><br>
-{% endblock %}
+<br><br><br> {% endblock %}
```
Comment (medhach) [inline templates/malaria/create_post.html:80]: Please leave an empty line at the end of every file.

File: templates/malaria/edit_post.html [hunk 5]
```diff
@@ -1,6 +1,4 @@
-{% include "infohub/index.html" %}
-
-{% block content %}
+{% include "infohub/index.html" %} {% block content %}
```
Comment (medhach) [inline templates/malaria/edit_post.html:1]: Why this change?

File: templates/pcsa/create_post.html [hunk 6]
```diff
@@ -48,5 +47,4 @@ <h2 class="text-center ">New Post</h2>
         </div>
     </form>
 </div>
-<br><br><br>
-{% endblock %}
+<br><br><br> {% endblock %}
```
Comment (medhach) [inline templates/pcsa/create_post.html:50]: Empty line at the end of the file

**Review by medhach — CHANGES_REQUESTED** [review]

**Comment by medhach** [comment]
@Tahirih-Natang I have a few concerns regarding your pull request (PR).
* Please revert the changes that you've made by auto-formatting the files. We don't need to merge them and make sure that you don't auto format the files in future as well.
* You have added so many changes in one PR. We strongly recommend that you add only one PR per change. for eg., one to add unit tests, one to change some HTML code, etc.
* If you want to add a new library that you think will help the project, first create an issue and we will discuss over whether it suits our project or not and then implement it. Can you explain why you've chosen to add the django-WYSIWYG library?
* You did a good job adding unit tests for the project, but please add inline-comments for what each test does.
Good job with commit messages. Keep them clear and understandable in future too. :)
Please make the requested changes and open separate PRs for each change. You can create separate PRs by creating separate branches for each change. Let me know if you need help on that.
Closing this PR for now.

File: pcsa_GHN/tests.py (hunk 2 above)
Comment (Tahirih-Natang) [reply pcsa_GHN/tests.py:33]: The test method is buggy, I am still to fix that.

File: templates/malaria/create_post.html (hunk 3 above)
Comment (Tahirih-Natang) [reply templates/malaria/create_post.html:1]: That is not a code change, it's just formatting. My text editor for some reason I don't know default to that format each time I save the file. I will fix that

File: templates/malaria/edit_post.html (hunk 5 above)
Comment (Tahirih-Natang) [reply templates/malaria/edit_post.html:1]: Still not a code change, It's just formatting which should not break anything

**Comment by Tahirih-Natang** [comment]
@medhach I thought adding a WYSIWYG to the forms will give users a better way of formatting their posts and making them appear the way they desire. 
Did you pull my changes and try to test them on your own system?
